src/main/java/com/laserfiche/repository/api/clients/EntriesClient.java

```diff
@@ -52,18 +54,19 @@ CompletableFuture<ODataValueOfIListOfFieldValue> assignFieldValues(String repoId
      * - Optionally sets metadata and electronic document component.
      * - Optional parameter: autoRename (default false). If an entry already exists with the given name, the entry will be automatically renamed. With this route, partial success is possible. The response returns multiple operation (entryCreate operation, setEdoc operation, setLinks operation, etc..) objects, which contain information about any errors that may have occurred during the creation. As long as the entryCreate operation succeeds, the entry will be created, even if all other operations fail.
      *
-     * @param requestBody   null
      * @param repoId        The requested repository ID.
      * @param parentEntryId The entry ID of the folder that the document will be created in.
      * @param fileName      The created document's file name.
      * @param autoRename    An optional query parameter used to indicate if the new document should be automatically
      *                      renamed if an entry already exists with the given name in the folder. The default value is false.
      * @param culture       An optional query parameter used to indicate the locale that should be used.
      *                      The value should be a standard language tag.
+     * @param file          The file that will be uploaded.
+     * @param requestBody   null
      * @return CompletableFuture<CreateEntryResult> The return value
      */
-    CompletableFuture<CreateEntryResult> importDocument(PostEntryWithEdocMetadataRequest requestBody, String repoId,
-            Integer parentEntryId, String fileName, Boolean autoRename, String culture);
+    CompletableFuture<CreateEntryResult> importDocument(String repoId, Integer parentEntryId, String fileName,
+            Boolean autoRename, String culture, File file, PostEntryWithEdocMetadataRequest requestBody);
```

Comment:
very minor, but should the `File file` parameter, be `File electronicDocument` to match swagger specs?

Reply:
Yes.